### PR TITLE
PRIME-2803 reflect changes at CareConnect

### PR DESCRIPTION
**Pull request by bergomi02**
Waiting for the content to be finalized!

### CODE REVIEW

File: prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.scss [hunk 1]
```diff
@@ -150,3 +150,9 @@ a.tab-link:visited {
 .mat-select-value {
   color: #ffffff !important;
 }
+
+.warning-text {
+  color: rgb(245, 123, 123);
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.scss:155]: Can this just be `red`?

File: prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts [hunk 2]
```diff
@@ -170,15 +173,41 @@ export class CareSettingPageComponent extends AbstractCommunitySiteRegistrationP
...
+            if (confirmation) {
+              return request$;
+            }
+            return EMPTY;
+          })
+        );
+    } else {
+      if (this.formState.json.careSettingCode === CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE) {
+        this.siteFormStateService.businessLicenceFormState.resetSiteId();
+      }
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts:204]: Not sure if I understand the need for `resetSiteId` but looking at previous logic, shouldn't `resetSiteId` be executed even if user has selected CareConnect?

File: prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts [hunk 3]
```diff
@@ -170,15 +173,41 @@ export class CareSettingPageComponent extends AbstractCommunitySiteRegistrationP
...
+
+    const sitePayload = this.siteFormStateService.json;
+    const request$ = this.siteResource.updateSite(sitePayload);
+
+
+    if (this.formState.json.careSettingCode === CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE
+      && this.vendorConfig.find(v => v.code === this.formState.vendorCode.value).name === 'CareConnect'
+    ) {
+      const data: DialogOptions = {
+        title: 'Save & Continue',
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts:185]: I think use of "&" is frowned on in PRIME labels and text ... change to `'Save and Continue confirmation'`

Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts:185]: Updated as suggested.

File: prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts (hunk 2 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts:204]: Yes, it was missed. The code is updated. FYI, I believe the code to reset the site id is for case where the care setting changes from community pharmacy to PCHP.

**Review by neophyte57 — APPROVED** [review]
Nicely done!

**Pull request merged**